src/App.jsx

```diff
@@ -10,6 +10,7 @@ import Login from './pages/Login/Login';
 import Signup from './pages/SignUp/SignUp';
 import About from './pages/About/About';
 import Team from './pages/Team/team';
+import Issue from "./pages/rama-issue/issue";
```

Comment:
This is not an appropriate page name. Your code is related to the "Requirements" page. The folder should be called "Requirements" and the file "Requirements.js"